Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -76,7 +77,28 @@ public void StartNextRound()
     public void EndActiveRound()
     {
         onRoundEnd?.Invoke();
-        print("aaaaand the winner issssss" + rounds.Last().winner);
-        SceneManager.LoadScene("Menu");
+
+        // TODO assign chips
+
+        CheckWinCondition();
+    }
+
+    private void CheckWinCondition()
+    {
+        var lastWinner = rounds.Last().winner;
+        var wins = rounds.Select(round => round.winner == lastWinner).Count();
```

Comment:
This seems to always count every round won no matter which player was last winner, which makes the player who wins the third round always be the match-winner. 🤔